frontend/src/ducks/ledgers.ts

```diff
@@ -141,6 +143,32 @@ export const fetchLedgersTransactionsHistoryAction = createAsyncThunk<
   },
 );
 
+export const fetchLedgersModule = createAsyncThunk<
+  FetchLedgerModuleResponse[],
+  void,
+  { rejectValue: RejectMessage; state: RootState }
+>("ledgers/fetchLedgersModule", async (_, { rejectWithValue }) => {
+  try {
+    const historyFilter = ledgerTransactionHistoryConfig["30D"];
+    const response = await fetch(
+      `/api/ledgers${historyFilter.endpointPrefix}${
+        networkConfig[Network.MAINNET].ledgerTransactionsHistorySuffix
+      }`,
+    );
+
+    const operations = await response.json();
+
+    const result = operations.data.map((operation: LedgerModuleItem) => ({
+      date: new Date(operation.end),
```

Comment:
we should use a serializable data for this. If you look at the console logs this throws an error:
```
react_devtools_backend.js:4026 A non-serializable value was detected in the state, in the path: `ledgers.ledgerModule.0.date`. Value: Thu Jun 23 2022 09:38:06 GMT-0700 (Pacific Daylight Time) 
```